deployment/dev/devenv.sh

```diff
@@ -3,8 +3,15 @@
 cd $(dirname $0)
 cd compose
 
-# Build local images first
-bazel run //:build-images
+if [ -z "$BUILD" ]; then
+  echo "Using existing images. To rebuild, run:"
+  echo ""
+  echo "  BUILD=1 deployment/dev/devenv.sh ..args"
+  echo ""
+else
+  # Build local images first
+  bazel run //:build-images
+fi
 
 docker-compose \
   -f core.compose.yml \
```

Comment:
Is this change of behavior documented in any Readme?

Reply:
It is printed out with `echo`!

Reply:
was just no intuitive at first glace, expected "devenv.sh build ..args" xD